developer_guides/debugability/perf-counters/index.rst

```diff
@@ -0,0 +1,62 @@
+.. _dbg-perf-counters:
+
+Performance Counters
+####################
+
+Firmware can be configured to trace performance counters for each processing
+component. Each performance trace entry include: 
+
+- component UUID
+
+- peak platform & cpu timer ticks consumed during component's copy processing
+
+.. note::
+  Performance timestamp macros use both the platform timer and cpu timer in case
+  the latter is not always-running-timer.
+
+Performance Counters usage
+**************************
+
+Currently performance counters can be only enabled statically during FW build in
+one of two ways:
+
+#. Select "Performance counter" from "Debug" menu using make menuconfig
+
+#. Add "CONFIG_PERFORMANCE_COUNTERS=y" to specific FW config (e.g. tigerlake_defconfig)
```

Comment:
Currently, you can only enable performance counters statically during FW build in
one of two ways:

- Select **Performance counter** from the **Debug** menu using ``make menuconfig``

- Add ``CONFIG_PERFORMANCE_COUNTERS=y`` to a specific FW configuration, for example, tigerlake_defconfig.

Reply:
Can we make it a bullet list? A numbered list implies that both steps should be taken...